### PR TITLE
added new logo

### PR DESCRIPTION
**Pull request by finlaylp**
Can't put anything dynamic in require so this is the easiest way to do it

### CODE REVIEW

File: constants/index.js [hunk 1]
```diff
@@ -45,6 +45,10 @@ export const COLORS = {
   'default': ['#CAB0FF', '#9D8DFF', '#6D8DFF', '#4457BC'],
   'aqua': ['#4EC9DA', '#65C6BE', '#7DD8FF', '#86DDB4'],
 };
+export const LOGOS = {
+  'default' : require('../../assets/logo.png'),
```
Comment (ctperry0301) [inline constants/index.js]: I think it should just be one ../ for these routes

Comment (finlaylp) [reply constants/index.js]: No cuz this is the path relative to Home.jsx, not index.js

Comment (jpoist97) [reply constants/index.js]: I think Cole is right here, when I try to run the app as is it gives an error, but when I switch it to just one ../ it runs as expected

Comment (finlaylp) [reply constants/index.js]: yeah he was right forgot to come back and update it

**Review by rohithdara — APPROVED** [review]

**Review by ctperry0301 — APPROVED** [review]
looks good

**Pull request merged**